fontc/src/lib.rs

```diff
@@ -969,6 +969,7 @@ mod tests {
             vec![
                 Tag::new(b"GDEF"),
                 Tag::new(b"GPOS"),
+                Tag::new(b"GSUB"),
```

Comment:
was this wanted?

Reply:
yes, I added features.fea to an existing test case and added a `liga` rule, so this now generates GSUB.